### PR TITLE
Feature EPL-875: add ean128 & mobile scan

### PR DESCRIPTION
**Pull request by JoaquinCorti**

### CODE REVIEW

File: docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md [hunk 1]
```diff
@@ -202,3 +202,76 @@ Then `./gradlew setup` must be executed.
 !!! warning
     It is mandatory to execute this step before installing the module. 
 
+
+## EAN 128
+
+### Overview
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md:208]: All modules have their Technical Aspects description and in this case only Overview and description of methods or classes are added.

File: docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md [hunk 2]
```diff
@@ -202,3 +202,76 @@ Then `./gradlew setup` must be executed.
...
+- Apache Commons Lang for String manipulations.
+- JSON and Jettison for JSON parsing.
+
+### SMFEANComponentProvider.java
+
+**Purpose**: It defines the global resources that are necessary for the operation of the components it provides.
+
+## Mobile Scan
+
+### Overview
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md:249]: All modules have their Technical Aspects description and in this case only Overview and description of methods or classes are added.

File: docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md:1]: EAN 128 and Mobile Scan modules must also be added to the bundle Overview.

**Review by valeg-etendo — CHANGES_REQUESTED** [review]

File: docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md [hunk 3]
```diff
@@ -202,3 +202,76 @@ Then `./gradlew setup` must be executed.
...
+
+## EAN 128
+
+### Overview
+
+It is a module for the transmission of information between the agents of the supply chain under the specifications of the Code 128 barcode. This, provide generic Utils for EAN128 Definition and parsing
+
+Is a Java-based module for managing EAN-128 barcodes in an Etendo ERP context. Below is an overview of what each file appears to do:
+
+### SelectorFieldPropertyDataSource.java
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/bundles/platform-extensions-bundle.md:214]: Is it necessary to describe the classes of the module?

**Review by valeg-etendo — CHANGES_REQUESTED** [review]

**Review by sjcuello — APPROVED** [review]

**Review by isaiasb-etendo — APPROVED** [review]

**Pull request merged**